feat: UILD-409: Write installation documentation for ui-linked-data module (#13)
folio-org/ui-ld-folio-wrapper: README.md

> 

Reviewer: Line 1 contains the text "ui-linked-data". It has to be changed to "ui-ld-folio-wrapper"

## Introduction

> This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module for handling operations with Linked data resources.

Reviewer: ```suggestion
This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module for performing operations on Linked data resources.
```

> This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module for handling operations with Linked data resources.

Reviewer: ```suggestion
This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module designed for performing operations on a library's Linked Data resources.
```

> This module acts as an adapter for [ui-linked-data](https://github.com/folio-org/ui-linked-data) module. Essentially, most of the functionality of this module is defined there.

Reviewer: This module serves as an adapter for the [ui-linked-data](https://github.com/folio-org/ui-linked-data) application. Essentially, most of the functionality of this module is defined in [ui-linked-data](https://github.com/folio-org/ui-linked-data). This module enables the [ui-linked-data](https://github.com/folio-org/ui-linked-data) application to run seamlessly within the FOLIO platform.

## Development

If you would like to run a development build of this module while making changes to the [ui-linked-data](https://github.com/folio-org/ui-linked-data) within it, there are a few steps to take:

1. Clone the [ui-linked-data](https://github.com/folio-org/ui-linked-data) module repository to your computer.
2. Within the cloned [ui-linked-data](https://github.com/folio-org/ui-linked-data) module directory, run:
    ```sh
    yarn link
    ```
3. Within this module's directory, run:
    ```sh
    yarn link @folio/linked-data
    ```
4. Perform the code changes you need.
5. Build the [ui-linked-data](https://github.com/folio-org/ui-linked-data) module as a library. The steps are described [here](https://github.com/folio-org/ui-linked-data?tab=readme-ov-file#as-an-embedded-application).
6. After the [ui-linked-data](https://github.com/folio-org/ui-linked-data) module finishes its build, the Stripes build should pick up the changes and reload itself.

## Additional information

> See the related [ui-linked-data](https://github.com/folio-org/ui-linked-data) module.

Reviewer: ```suggestion
## Additional information
See the related [ui-linked-data](https://github.com/folio-org/ui-linked-data) module.
Backend module corresponding to this UI module: [mod-linked-data](https://github.com/folio-org/mod-linked-data)
```

> Other [modules](https://dev.folio.org/source-code/#client-side).

Reviewer: Eventhough the hyperlink says "modules", the link is not pointing to any modules.

Author: This is a common section for UI modules. Check here: https://github.com/folio-org/ui-users/?tab=readme-ov-file#additional-information or here: https://github.com/folio-org/ui-users/?tab=readme-ov-file#additional-information

## Introduction

This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module for handling operations with Linked data resources.

This module acts as an adapter for [ui-linked-data](https://github.com/folio-org/ui-linked-data) module. Essentially, most of the functionality of this module is defined there.

## Development

If you would like to run a development build of this module while making changes to the [ui-linked-data](https://github.com/folio-org/ui-linked-data) within it, there are a few steps to take:

> 0. 

Reviewer 2: Do we need this empty step?
